Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Person.hs

```diff
@@ -117,7 +120,9 @@ listPerson _ mbSearchString mbLimit mbOffset mbPersonId = do
   res <- forM personAndRoleList $ \(encPerson, role, merchantAccessList) -> do
     decPerson <- decrypt encPerson
     pure $ DP.makePersonAPIEntity decPerson role merchantAccessList
-  pure $ ListPersonRes res
+  let count = length res
+  let summary = Summary {totalCount = count, count}
```

Comment:
Why do we need  this Summary, its just length of response list, no useful info

Reply:
yeah Roman, not useful. But our UI framework requires response like this only. So, we have to send in this format else it can break in UI.